tests/dummy/app/templates/components/attachment-example.hbs

```diff
@@ -304,6 +305,22 @@
                           {{/power-select}}"
       </centered>
     </hbox>
+    {{#unless isConfiguringTooltip}}
```

Comment:
Is there a particular reason to hide this when people are playing with `attach-tooltip`?

Also, let's keep this alphabetically sorted within the list of configurables :)